plerkle_messenger/src/redis/redis_messenger.rs

```diff
@@ -128,7 +191,7 @@ impl RedisMessenger {
             // Get data from map.
 
             let bytes = match data {
-                Value::Data(bytes) => bytes,
+                Value::BulkString(bytes) => bytes,
```

Comment:
What causes this change to be needed?  I see its handled as `Value::BulkString` in `recv` as well but I don't see a config change for this.  Will this be a breaking change for someone with existing redis config?  Should we match both `Value::Data` and `Value::BulkString`?

Reply:
Redis crate update causes these changes. It was updated from 0.22.3 to 0.27.2, so Value enum was slightly changed. In previous version [Data was representing arbitrary binary data](https://docs.rs/redis/0.22.3/redis/enum.Value.html#variant.Data) and now [BulkString is representing it](https://docs.rs/redis/0.27.2/redis/enum.Value.html#variant.BulkString).
Don't think that we should mathch it somehow because internal data type was not changed, it's still `Vec<u8>`.

Reply:
ok that should be fine then as it sounds like not a breaking change for users of previous versions of plerkle messenger.